tsconfig.cjs.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
you can avoid a lot of duplication here by having a root `tsconfig.json` and using the [`extends`](https://www.typescriptlang.org/tsconfig#extends) field for each module specific tsconfig